content/blog/insights-resources-v2/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Updated Resources View"
+date: 2024-10-02T14:35:35-07:00
+draft: false
+meta_desc: The updated Resources page provides a new user experience for creating custom views that help you gain an understanding and insights into your infrastructure.
+
+meta_image: meta.png
+authors:
+    - craig-symonds
+tags:
+    - insights
+    - resources
+---
+
+Today we are excited to release an updated version of the Pulumi Insights Resources page that provides a new user experience for grouping, filtering and sorting, and enables you to create custom views to help you gain additional understanding and insights of your infrastructure.
+
+<!--more-->
+
+* **Grouping** provides a rich model for factoring your resources into logical groupings to help you gain insights over your infrastructure and plan actions in incremental groupings. To create a group, drag the column header to the grouping control. Note that you can create hierarchical groups by adding multiple columns and even rearrange the order of the groups.
+* **Filtering** allows you to quickly find specific resources or groups of resources based on different attributes of the resource. Filtering can be done by selecting the filter icon in the column header for the column you want to filter by. This will present a drop down that enables you to select, or enter, the values to filter by. Adding column filters updates the Resource Search field to reflect the filters selected, in this way, the column filtering can be used to help create Resource Queries that can be reused in other contexts.
+* **Sorting** is done in the standard way by clicking on the column header.
+
+{{< video title="Grouping, Filtering, Sorting" src="grouping-sorting-filtering.mp4" autoplay="true" loop="true" >}}
+
+The updated Resources page enables you to customize the view of your resources by enabling you to control the columns of the table. With the table actions menu you can add and remove columns as well as auto size each of the columns in the table to fit the current content. Like previous versions of the Resources page, this menu also enables you to export the Resources as a CSV file for integration into external tools.
```

Comment:
maybe add a heading before each of these little paragraphs that have their own video?

Reply:
Fixed - headers added.